lib/internal/http2/compat.js

```diff
@@ -251,6 +251,7 @@ function onStreamTimeout(kind) {
 
 class Http2ServerRequest extends Readable {
   constructor(stream, headers, options, rawHeaders) {
+    if (!options) options = {};
```

Comment:
Just to make sure I’m not missing anything – how does this change relate to the others?